sbin/common/common.sh

```diff
@@ -147,16 +148,16 @@ createOpenJDKArchive()
   else
       # Create archive with UID/GID 0 for root if using GNU tar
       if tar --version 2>&1 | grep GNU > /dev/null; then
-          time tar -cf - --owner=root --group=root "${repoDir}"/ | GZIP=-9 $COMPRESS -c > $fileName.tar.gz
+          time tar -cf - --owner=root --group=root "${repoDir}"/ | GZIP=-9 $COMPRESS -c > "$fileName.tar.gz"
       else
-          time tar -cf - "${repoDir}"/ | GZIP=-9 $COMPRESS -c > $fileName.tar.gz
+          time tar -cf - "${repoDir}"/ | GZIP=-9 $COMPRESS -c > "$fileName.tar.gz"
       fi
   fi
 }
 
 function setBootJdk() {
   # Stops setting the bootJDK on the host machine when running docker-build
-  if [ "${BUILD_CONFIG[DOCKER]}" != "docker" ] || ([ "${BUILD_CONFIG[DOCKER]}" == "docker" ] && [ "${BUILD_CONFIG[DOCKER_FILE_PATH]}" != "" ]) ; then
+  if [ "${BUILD_CONFIG[DOCKER]}" != "docker" ] || { [ "${BUILD_CONFIG[DOCKER]}" == "docker" ] && [ "${BUILD_CONFIG[DOCKER_FILE_PATH]}" != "" ]; } ; then
```

Comment:
I'm worried about this change? If I run
```
if false || { false && true }; then echo true; else echo false; fi
```
but
```
if false || ( false && true ); then echo true; else echo false; fi
```
works

Reply:
For reference, [SC2235](https://github.com/koalaman/shellcheck/wiki/SC2235) is the directive. It's mostly to speed up the processing time. I am confused what the problem is, both statements and the alternative syntax
```bash
if false || { false && true; }; then echo true; else echo false; fi
```
amount to the same result for me?

Reply:
Hmmm whichever shell I was using at the time definitely wouldn't work with that one

Reply:
The first ";" is important, ie.:
```
if false || { false && true; } ; then
```

Reply:
I think
```bash
if false || { false && true; } ; then
```
would produce a syntax error because of the whitespace after `}`